pkg/daemon/ceph/osd/volume.go

```diff
@@ -115,7 +117,17 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 			if a.cluster.CephVersion.IsAtLeast(cephVolumeRawModeMinCephVersion) && !lvBackedPV {
 				// For block mode
 				block = fmt.Sprintf("/mnt/%s", a.nodeName)
-				rawOsds, err = getCephVolumeRawOSDs(context, a.cluster.Name, a.cluster.FSID, block, lvBackedPV)
+
+				// This is hard to determine a potential metadata device here
+				// Also, I don't think (leseb) this code we have run in this condition
+				// I tried several things:
+				//    * evict a node, osd moves, the prepare job was never relaunched ever because we check for the osd deployment and skip the prepare
+				//    * restarted the operator, again the prepare job was not re-run
```

Comment:
Yes, it's still unclear to me, I need to trace back all the commits because that code looks unreachable to me too. I guess I'm "afraid" to forget a case we solved and introducing a regression.

So I'd like to keep it around a bit so we are sure, wdyt?

Reply:
Sure let's set `metadataBlock=""`. I think we could simplify the comment though. Not sure we need the details of everything you tested.